libs/filabridge/include/private/filament/Variant.h

```diff
@@ -206,85 +212,15 @@ struct Variant {
         return Variant(key & rhs);
     }
 
+    static Variant filterVariant(
+            Variant variant, UserVariantFilterMask filterMask) noexcept;
+
 private:
     void set(bool v, type_t mask) noexcept {
         key = (key & ~mask) | (v ? mask : type_t(0));
     }
 };
 
-namespace details {
-
-// compile time sanity-check tests
```

Comment:
Could you move `isFragmentVariant` and `isVertexVariant` to the cpp as well, since they're only called by these compile-time checks?

Reply:
well they are for the moment, but I think they could be part of the Variant API.